llms/mlx_lm/convert.py

```diff
@@ -94,11 +94,10 @@ def quantize_model(
     model, _ = load(hf_path)
     model.load_weights(list(weights.items()))
 
-    nn.QuantizedLinear.quantize_module(model, q_group_size, q_bits)
-    quantized_config["quantization"] = {
-        "group_size": q_group_size,
-        "bits": q_bits,
-    }
+    nn.QuantizedLinear.quantize_module(
+        model, q_group_size, q_bits, linear_class_predicate=linear_class_predicate
```

Comment:
Add the linear_class_predicate so that we don't quantize any layer with dimensions that can't be divided by 32.